docs/introduction/get-started/p2p-public-note-txs.md

```diff
@@ -0,0 +1,68 @@
+In this section, we show you how to execute transactions and send funds to another account using the Miden client through public notes. 
+
+!!! important "Prerequisite steps"
+    - You should have already followed all previous sections.
+    - You should have *not* reset the state of your local client. 
+
+## Create a second client
+
+!!! tip
+      Remember to use the [Miden client documentation](https://docs.polygon.technology/miden/miden-client/cli-reference/) for clarifications.
+
+This is an alternative to the private offchain P2P transactions article. For this tutorial, we will utilize two different clients to simulate two different remote users who don't share local state. To do this, we can have two terminals with their own state (using their own `miden-client.toml`). One terminal can run the client from the previous article, and a new terminal can run a new client.
```

Comment:
Can you maybe explain how a user can create a new client instance? 

To me it looks like I need to 

- open a new terminal window
- create and go to a new folder, e.g., `mkdir miden-client-2` and `cd miden-client-2` 
- run `miden-client init` agin